unittests/Driver/DependencyGraphTests.cpp

```diff
@@ -773,13 +763,13 @@ TEST(DependencyGraph, ChainedExternalReverse) {
 TEST(DependencyGraph, ChainedExternalPreMarked) {
   DependencyGraph<uintptr_t> graph;
 
-  EXPECT_EQ(graph.loadFromString(0,
-                                 "depends-external: [/foo]\n"
-                                 "provides-top-level: [a]"),
+  EXPECT_EQ(graph.loadFromString(0, externalDependencies +
+                                        ": [/foo]\n" topLevelProviders +
+                                        ": [a]"),
             LoadResult::UpToDate);
-  EXPECT_EQ(graph.loadFromString(1,
-                                 "depends-external: [/bar]\n"
-                                 "depends-top-level: [a]"),
+  EXPECT_EQ(graph.loadFromString(1, externalDependencies +
+                                        ": [/bar]\n" topLevelDependencies +
```

Comment:
I assume the work on this file you mentioned includes adding the necessary `+` calls.

Reply:
Yes, check out the latest push and let me know. (Haven't pushed the other fixes yet.) Tx!